client/src/shared/components/Layout-learning/SideBar.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link';
```

Comment:
- incorrect file folder structure. inquire this to members
![image](https://user-images.githubusercontent.com/73863685/222644812-b6de26e0-910c-47fb-8a03-ee5d87eaf466.png)